interactive/printer.go

```diff
@@ -14,20 +12,17 @@ func (l byLength) Len() int           { return len(l) }
 func (l byLength) Swap(i, j int)      { l[i], l[j] = l[j], l[i] }
 func (l byLength) Less(i, j int) bool { return len(l[i]) > len(l[j]) }
 
-// QuoteMarkedFiles takes files from the map and returns slice of qoted file paths
-func QuoteMarkedFiles(markedFiles map[*core.File]struct{}) []string {
-	quotedFiles := make([]string, len(markedFiles))
-	i := 0
-	for file := range markedFiles {
+// FilesAsSlice takes files from the map and returns a sorted slice of file paths.
+// Each file path is quoted if 'quote' is set to true.
```

Comment:
I believe this is not true, we are not going to be quoting the filenames.

Reply:
Whoopsie, too much copy & paste 😊.
Updated.